test/extended/testdata/cli/pod-with-two-containers.yaml

```diff
@@ -0,0 +1,33 @@
+kind: Pod
+apiVersion: v1
+metadata:
+  name: doublecontainers
+  labels:
+    name: hello-openshift
+spec:
+  containers:
+  - name: hello-openshift
+    image: aosqe/hello-openshift
```

Comment:
I'd prefer we stick what's already available and used everywhere so `openshift/hello-openshift` please.